### PR TITLE
Fdse 1015 - Merge 2nd

### PR DESCRIPTION
**Pull request by pmilano1**

### CODE REVIEW

File: rubrik_polaris/lib/accounts.py [hunk 1]
```diff
@@ -323,13 +352,24 @@ def _destroy_aws_stack(self, stack_region, stack_name, profile='', aws_id=None,
...
-    Arguments:
-        all {bool} -- If true import all available profiles (Default: False)
-        profiles {list} -- List of explicit profiles to add
-        aws_access_key_id {str} -- AWS Access Key ID
-        aws_secret_access_key {str} -- AWS Access Key Secret
+    """Remove AWS account from Polaris
+
+    Args:
+        profiles (list): Optional list of local profile names to add to Polaris
+        all (bool): Optional set true to import all locally configured profiles to Polaris
```
Comment (johan3141592) [inline rubrik_polaris/lib/accounts.py:359]: Says add/import, should probably be delete or remove.

File: rubrik_polaris/lib/accounts.py [hunk 2]
```diff
@@ -485,24 +532,37 @@ def add_project_gcp(self, service_account_auth_key_file=None, gcp_native_project
 
 
 def delete_project_gcp(self, gcp_native_project_id=None, delete_snapshots=False):
+    """Remove GCP project from Polaris
+
+    Args:
+        gcp_native_project_id (str): Project_Id of GCP Project to add
```
Comment (johan3141592) [inline rubrik_polaris/lib/accounts.py:538]: Says add, should probably be delete or remove.

File: rubrik_polaris/lib/accounts.py (hunk 1 above)
Comment (pmilano1) [reply rubrik_polaris/lib/accounts.py:359]: Fixing in 1016

File: rubrik_polaris/lib/accounts.py (hunk 2 above)
Comment (pmilano1) [reply rubrik_polaris/lib/accounts.py:538]: Fixing in 1016

**Review by rzferchichi — APPROVED** [review]

**Pull request merged**